nroam-backlinks.el

```diff
@@ -120,12 +120,13 @@ When nil, LEVEL defaults to 3."
   (with-current-buffer (nroam-backlinks--work-buffer)
     (insert-file-contents file nil nil nil 'replace)
     (goto-char point)
-    (let* ((elt (org-element-at-point))
-           (begin (org-element-property :begin elt))
-           (end (org-element-property :end elt))
-           (type (org-element-type elt))
-           (outline (org-get-outline-path))
-           (full-outline (and outline (org-get-outline-path 'with-self))))
+    (let ((elt (org-element-at-point))
```

Comment:
Why replace `let*` with `let`? `elt` is used below in other let forms, so this breaks.
FWY I just fixed it in master.

Reply:
Ah, ok. Thanks. Yeah, as mentioned a few seconds ago in the comment above I messed up.

But then I know it's fixed :)